src/js/utils/normalize.ts

```diff
@@ -0,0 +1,15 @@
+import { normalize } from '@sentry/utils';
+
+/**
+ * Converts any input into a valid record with string keys.
+ */
+export function convertToNormalizedObject(wat: unknown): Record<string, unknown> {
```

Comment:
```suggestion
export function convertToNormalizedObject(data: unknown): Record<string, unknown> {
```

M: Always prefer more meaningful names.

Reply:
It's used in JS a lot to name variables that you don't know what will be inside. 
https://github.com/getsentry/sentry-javascript/blob/2aa4e94b036675245290596884959e06dcced044/packages/utils/src/is.ts#L16

Reply:
That is fine, but it always comes from `breadcrumb.data`, so lets call `data`?